### PR TITLE
feat: show theme picker in menu

### PR DESCRIPTION
**Pull request by aoneill01**
### Resolves

- Resolves [ENA-248](https://scratchfoundation.atlassian.net/browse/ENA-248)

### Proposed Changes

- Design specs linked in [ENA-264](https://scratchfoundation.atlassian.net/browse/ENA-264)
- Add a theme picker to the menu bar
- Implement design requirements, including:
  - Add label to language and theme picker showing the currently selected value
  - Hide the label when the screen is <= 1024 pixels in width
  - Move Tutorials menu item to the right
- Persist the selected theme
  - Store it in a cookie
  - Load the theme when page is refreshed
  - If the user selects a theme that matches the operating system preferences (high contrast on/off), treat that as preferring to use system preferences.

Expanded menu and shortened language value:

Screenshot 2023-03-13 at 2 24 14 PM

Right-to-left language:

Screenshot 2023-03-13 at 2 24 33 PM

Icon only when <= 1024 px:

Screenshot 2023-03-13 at 2 25 26 PM

### Reason for Changes

As part of the color theme epic, users will want to select a color theme for the editor.

### Test Coverage

- Integration tests to interact with the theme picker and verify style changes
- Unit tests for persistence helpers

### Browser Coverage
Check the OS/browser combinations tested (At least 2)

Mac
 * [x] Chrome 
 * [x] Firefox 
 * [x] Safari
 
Windows
 * [ ] Chrome 
 * [ ] Firefox 
 * [ ] Edge
 
Chromebook
 * [ ] Chrome
 
iPad
* [ ] Safari

Android Tablet
* [ ] Chrome

[ENA-248]: https://scratchfoundation.atlassian.net/browse/ENA-248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ENA-264]: https://scratchfoundation.atlassian.net/browse/ENA-264?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ilikecereal1legacy** [comment]
why is there no dark mode? tested on iPad Safari

**Comment by mxmou** [comment]
> why is there no dark mode? tested on iPad Safari

Probably because it isn't finished yet.

File: src/components/menu-bar/menu-bar.css [hunk 1]
```diff
@@ -52,16 +52,28 @@
 .language-icon {
     height:  1.5rem;
     vertical-align: middle;
+    margin: 0 .125rem 0 0;
 }
 
-.language-caret {
-    margin: 0 .125rem;
+[dir="rtl"] .language-icon {
+    margin: 0 0 0 .125rem;
```
Comment (aoneill01) [inline src/components/menu-bar/menu-bar.css:59]: I wish we could use `margin-inline-[start|end]` instead. But we need to support browser versions [without support](https://caniuse.com/?search=margin-inline).

https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline

**Pull request merged**